Makefile

```
@@ -1,6 +1,6 @@
 CC         = gcc
 JAVAH      = javah
-MYCFLAGS   = -c -Wall -fPIC $(EXTRA_CFLAGS) `pkg-config atk-bridge-2.0 --libs`
+MYCFLAGS   = -c -Wall -fPIC $(EXTRA_CFLAGS) `pkg-config --libs atk-bridge-2.0`
```

Maybe --cflags instead of --libs? Though I see you add it below. Is this needed then?

---

I've update again please give me a feedback
```suggestion
MYCFLAGS   = -c -Wall -fPIC $(EXTRA_CFLAGS) `pkg-config atk-bridge-2.0 --libs`
```
or is more correct without --libs.

---

Why are you doing this change? Also, "test" isn't exactly a good me commit message.